go/obscuronode/host/host.go

```diff
@@ -470,43 +488,86 @@ func (a *Node) handleStoreSecretTx(t *obscurocommon.L1RespondSecretTx) bool {
 	return true
 }
 
-func (a *Node) checkForSharedSecretRequests(block obscurocommon.EncodedBlock) {
```

Comment:
GitHub made a real pig's ear of this, but the logic in `checkForSharedSecretRequests` was moved more or less intact to `processSharedSecretRequest`.